pydantic/schema.py

```diff
@@ -69,6 +70,7 @@ def schema(
     title: Optional[str] = None,
     description: Optional[str] = None,
     ref_prefix: Optional[str] = None,
+    ref_template: Optional[string.Template] = None,
```

Comment:
With no extra handling...
1. If `str` is passed instead of a `string.Template` an `AttributeError` will be raised.
   `AttributeError: 'str' object has no attribute 'substitute'`
2. If `string.Template` is passed that doesn't have the `$model_name` identifier a `KeyError` will be raised.
     `KeyError: 'foo_bar'`

https://docs.python.org/3/library/string.html#template-strings